isotovideo

```diff
@@ -89,13 +133,27 @@ sub _get_version_string () {
     return "Current version is $thisversion [interface v$OpenQA::Isotovideo::Interface::version]";
 }
 
+sub _exit_code_from_test_results () {
+    my @results = glob(path(bmwqemu::result_dir(), "/result-*.json"));
+    return EXIT_STATUS_ERR_NO_TESTS if scalar @results == 0;
```

Comment:
`scalar` is implicit here with the `==` operator.

Reply:
TIL. Changed in 0df7e176